gcsfs/tests/conftest.py

```diff
@@ -115,5 +116,30 @@ def gcs(gcs_factory, populate=True):
     finally:
         try:
             gcs.rm(gcs.find(TEST_BUCKET))
+            gcs.rm(TEST_BUCKET)
```

Comment:
We need to remove the empty bucket when we are done with it since depending on the next test, we may need to re-create the bucket with different configuration options (`versioning.enabled` = true/false)

Reply:
+1